File: ruby/managing-navigation/GetArticlesDepth.rb
# DocSection: managing_navigation_articles_depth
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.item('root_navigation_item')
               .depth(5)
               .execute do |response|
                 item = response.item
               end
# EndDocSection

File: php/managing-navigation/GetArticlesDepth.php
// DocSection: managing_navigation_articles_depth
<?php
// Tip: Find more about PHP SDKs at https://docs.kontent.ai/php

// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kentico\Kontent\Delivery\DeliveryClient;

$client = new DeliveryClient('24246f25-946d-4ab4-b170-fadef22fe7b2');

$item = $client->getItem('root_navigation_item', (new QueryParams())
            ->depth(5));
// EndDocSection